docs/configuration/arguments.md

```diff
@@ -385,21 +385,6 @@ It also talks to a KVStore and has it's own copies of the same flags used by the
    Where you don't want to cache every chunk written by ingesters, but you do want to take advantage of chunk write deduplication, this option will make ingesters write a placeholder to the cache for each chunk.
    Make sure you configure ingesters with a different cache to queriers, which need the whole value.
 
-#### WAL
```

Comment:
Already covered by the dedicated doc https://cortexmetrics.io/docs/chunks-storage/ingesters-with-wal/